Frontend/ui-customer/src/Screens/Types.purs

```diff
@@ -570,14 +570,21 @@ type Contact = {
      phoneNo :: String
 }
 
+data RateCardType = ViewStart | ViewDriverAddition | ViewFareUpdate
```

Comment:
Change the following
 ViewStart -> Default
 ViewDriverAddition -> DriverAddition
 ViewFareUpdate -> FareUpdate